### PR TITLE
Vi Keybind and insert urwid_readline to Pipenv

### PR DESCRIPTION
**Pull request by arthurhenrique**

### CODE REVIEW

File: sclack/components.py [hunk 1]
```diff
@@ -200,7 +200,7 @@ def keypress(self, size, key):
         if key == keymap['go_to_sidebar']:
             self.header.restore_topic()
             urwid.emit_signal(self, 'go_to_sidebar')
-            return True
+            return key
```
Comment (haskellcamargo) [inline sclack/components.py]: Is this really necessary?

Comment (arthurhenrique) [reply sclack/components.py]: this line `urwid.emit_signal(self, 'go_to_sidebar')` return the `key` so `return True` neither `return key` is really necessary...

Comment (haskellcamargo) [reply sclack/components.py]: We should keep `return True` for 2 main reasons:
- Keeping them standard
- Avoiding the propagation of the event to the top-components (when they exist). That's a common source of bugs where some events are handled by two components.

Comment (haskellcamargo) [reply sclack/components.py]: It happened on #5 too.

Comment (arthurhenrique) [reply sclack/components.py]: Gotcha! I'll find another solution...

Comment (arthurhenrique) [reply sclack/components.py]: I remove this if.

**Pull request merged**